loki/transform/fortran_c_transform.py

```diff
@@ -50,8 +50,9 @@ class FortranCTransformation(Transformation):
     # Set of standard module names that have no C equivalent
     __fortran_intrinsic_modules = ['ISO_FORTRAN_ENV', 'ISO_C_BINDING']
 
-    def __init__(self, header_modules=None, inline_elementals=True):
+    def __init__(self, header_modules=None, inline_elementals=True, path=None):
```

Comment:
Would you mind adding a docstring for the `path` parameter? (And, while you're at it, also `header_modules` and `inline_elementals`?)